### PR TITLE
layouts

### PR DESCRIPTION
**Pull request by jurajvandor**
main activity layout (should be changed to main fragment) created
settings layout created (snooze time not working - crashes)

### CODE REVIEW

File: app/src/main/java/com/project/pv239/customtimealarm/Activities/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu, menu);
```
Comment (Kuchto47) [inline app/src/main/java/com/project/pv239/customtimealarm/Activities/MainActivity.java]: menu menu menu :D :D :D :D

File: app/src/main/java/com/project/pv239/customtimealarm/Activities/SettingsActivity.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import android.view.MenuItem;
+
+import com.project.pv239.customtimealarm.Fragments.SettingsFragment;
+
+public class SettingsActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        if (savedInstanceState == null) {
```
Comment (Kuchto47) [inline app/src/main/java/com/project/pv239/customtimealarm/Activities/SettingsActivity.java]: len moja otazka (neovaldam javu z pamati) - ide napisat !savedInstanceState? (namiesto blabla == null ze napises iba !blabla) :))

File: app/src/main/res/xml/settings.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            android:title="@string/bedtime" />
+        <Preference
+            android:key="bedtime_time"
+            android:title="@string/time_to_sleep" />
+
+        <SwitchPreference
+            android:title="@string/notification_GTFO"
+            android:key="GTFO"/>
+        <ListPreference
+            android:title="@string/snooze_time"/>
```
Comment (Kuchto47) [inline app/src/main/res/xml/settings.xml]: https://developer.android.com/reference/android/preference/ListPreference.html

chcelo by to tam asi pridat android:entries na ten array 1..15 co si vytvoril a asi nieco v kode..potom to mozno ani padat nebude... :D

File: app/src/main/java/com/project/pv239/customtimealarm/Activities/SettingsActivity.java (hunk 2 above)
Comment (jurajvandor) [reply app/src/main/java/com/project/pv239/customtimealarm/Activities/SettingsActivity.java]: podla mna to nejde a pride mi to takto aj citatelnejsie to == null:)

File: app/src/main/res/xml/settings.xml (hunk 3 above)
Comment (jurajvandor) [reply app/src/main/res/xml/settings.xml]: hej hej viem :D len uz sa mi to nechcelo riesiet tak som to nechal tak

**Pull request merged**